### ISSUE TITLE
Error on using storage

### ISSUE THREAD

**Issue by trickyc0d3r** [psperber/redux-persist-electron-storage#14]
Hi! I'm facing this error `Uncaught TypeError: fs.existsSync is not a function
    at getElectronPath (index.js:7)
    at Object.parcelRequire.node_modules/electron/index.js.fs (index.js:18)
    at newRequire (src.a2b27638.js:47)
    at localRequire (src.a2b27638.js:53)
    at Object.parcelRequire.node_modules/electron-store/index.js.path (index.js:3)
    at newRequire (src.a2b27638.js:47)
    at localRequire (src.a2b27638.js:53)
    at Object.parcelRequire.node_modules/redux-persist-electron-storage/lib/index.js.electron-store (index.js:2)
    at newRequire (src.a2b27638.js:47)
    at localRequire (src.a2b27638.js:53)` while using your library. Have any solutions?

**Comment by zcmgyu** [comment]
Same here

**Comment by FarisCode** [comment]
Add `nodeIntegration:true` in webPreferences.

`mainWindow = new BrowserWindow({
    title: "sampleApp"
    width: 300,
    height: 450,
    webPreferences: {
      nodeIntegration: true
    }
  });`

use `window.require('electron-store');`

**Comment by trickyc0d3r** [comment]
Yes, with integration it' working, but what to do if I need to turn all such things off for security reasons.
